src/index.js

```diff
@@ -1,32 +1,50 @@
-
 let todos = [
-    {
-        completed: false,
-        description: "Take Jim to the hair salon"
-    },
-    {
-        completed: true,
-        description: "Drop off wedding invitation at mailbox"
-    },
-    {
-        completed: false,
-        description: "Pick up the cake"
-    },
-    {
-        completed: false,
-        description: "Call the caterers"
-    }
+  {
+    completed: false,
+    description: "Take Jim to the hair salon",
+  },
+  {
+    completed: true,
+    description: "Drop off wedding invitation at mailbox",
+  },
+  {
+    completed: false,
+    description: "Pick up the cake",
+  },
+  {
+    completed: false,
+    description: "Call the caterers",
+  },
 ];
 
 function renderTodoApp() {
-    let app = document.querySelector("#app")
-    let h1 = document.createElement("h1")
-    // your code here
+  let app = document.querySelector("#app");
+  let h1 = document.createElement("h1");
+  // your code here
+  let list = document.createElement("ul");
 
-    h1.innerText = "Todo List"
+  for (let item of todos) {
+    li = document.createElement("li");
+    li.style.listStyle = "none";
+    li.innerHTML = `
+      <input type="checkbox" name="item" id="item" />
+      <label for="item">${item.description}</label>
+      `;
+    if (item.completed === true) {
```

Comment:
this is fine, but not very DRY. Since the only thing you are changing with the conditional here is the `checked` property on the input, you could check if the `item.completed === true` and just assign a variable to either `"checked"` or an empty string `""` and add it right on to line 30 above like so:

`let isChecked = item.completed ?  "checked" :  "";`
`li.innerHTML = 
    `<input isChecked type="checkbox" name="item" id="item" />`